inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp

```diff
@@ -733,45 +993,20 @@ struct jit_uni_reduce_kernel_f32 : public jit_uni_reduce_kernel, public jit_gene
             vextractf32x4(xmm_aux3, zmm_dst, 3);
             horiz_ps(xmm_aux2, xmm_aux3);
             horiz_ps(xmm_aux1, xmm_aux2);
-            load_embedded_horiz_store(xmm_aux1, dst_dt);
+            horiz_store(xmm_aux1, dst_dt, load_embedded);
         }
     }
 
-    inline void load_embedded_horiz_store(Xbyak::Xmm xmm_dst, memory::data_type dst_dt) {
+    inline void horiz_store(Xbyak::Xmm xmm_dst, memory::data_type dst_dt, bool load_embedded) {
```

Comment:
could you please update to uni_xxx for this function.

Reply:
Fixed. Thanks Chenhu!